docs/course/course-customization.md

```diff
@@ -15,6 +15,11 @@ The website is setup to be easy to run automatically through [GitHub](http://git
 
 - Update `# Setup` information in `_config.yml` in the main directory for proper
   site rendering.
+  - You must `push` this change to your repository to build and browse your 
+    forked version. 
+  - If you choose to keep the "Data Carpentry" name in your `title:`, please
+    [contact us via email ({{ site.email }})](mailto:{{ site.email }}) 
```

Comment:
I suggest moving the line "   - In a few minutes you should be able to see a copy of the site at:
     `https://yourusername.github.io/semester-biology/`" below the new line about pusing the change.

I understood it to mean "a few minutes after doing the fork you should be able to see".